### PR TITLE
Add function to get ip address of VM

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

**Comment by katcipis** [comment]
I also removed the CI that is failing and leaking resources

File: azure/vm.sh [hunk 1]
```diff
@@ -136,3 +136,10 @@ fn azure_vm_delete(name, group) {
 			--resource-group $group
 	)
 }
+
+fn azure_vm_get_ip_address(name, group, iface_index, ip_index) {
+        info <= azure vm list-ip-address $group --json
+        #echo $ips
+        ip <= echo $info | jq -r ".[0].networkProfile.networkInterfaces[" + $iface_index + "].expanded.ipConfigurations[" + $ip_index + "].privateIPAddress"
```
Comment (kamilash) [inline azure/vm.sh]: :astonished:

**Comment by kamilash** [comment]
Nice!

File: azure/vm.sh (hunk 1 above)
Comment (matheusvill) [inline azure/vm.sh]: :astonished:

Comment (katcipis) [inline azure/vm.sh]: auheuaheuaeau pretty simple right ? @tiago4orion feel the love for some maps + JSON support here :heart:

Comment (tiago4orion) [inline azure/vm.sh]: =)

**Pull request merged**